pkg/arvo/app/btc-node-hook.hoon

```diff
@@ -0,0 +1,354 @@
+          ~&  btc-resp
+          ~
+      ::
+          :: %abandon-transaction
+          :: %abort-rescan
+          :: %add-multisig-address
+          :: %backup-wallet
+          :: %bump-fee
+          %create-wallet
+        ^-  (list card)
+        =/  btc-store-req=btc-node-store-action
+          :+  %add-wallet   name.btc-resp
+          ?:(=('' warning.btc-resp) ~ (some warning.btc-resp))
+        [(btc-node-store-poke /store btc-store-req)]~
+      ::
+          :: %dump-privkey
+          :: %dump-wallet
+          :: %encrypt-wallet
+          :: %get-addresses-by-label
+      ::
+          %get-address-info
+        ^-  (list card)
+        ~&  [%address-info +.btc-resp]
+        ~
+      ::
+          %get-balance
+        ^-  (list card)
+        :_  ~
+        :*  %pass  /  %arvo  %d  %flog
+            %text  "amount={(trip +.btc-resp)}"
+        ==
+      ::
+          :: %get-balance
+          :: %get-new-address
+          :: %get-raw-change-address
+          :: %get-received-by-address
+          :: %get-received-by-label
+          :: %get-transaction
+          :: %get-unconfirmed-balance
+      ::
+          %get-wallet-info
+        ^-  (list card)
+        =/  btc-store-req=btc-node-store-action
+          [%update-wallet wallet-name.btc-resp +>:btc-resp]
+        [(btc-node-store-poke /update btc-store-req)]~
+      ::
+          :: %import-address
+          :: %import-multi
+          :: %import-privkey
+          :: %import-pruned-funds
+          :: %import-pubkey
+          :: %import-wallet
+          :: %key-pool-refill
+          :: %list-address-groupings
+          :: %list-labels
+          :: %list-lock-unspent
+          :: %list-received-by-address
+          :: %list-received-by-label
+          :: %lists-in-ceblock
+      ::
+          %list-transactions
+        ^-  (list card)
+        ~&  [%transactions +.btc-resp]
+        ~
+      ::
+          :: %list-unspent
+          :: %list-wallet-dir
+      ::
+          %list-wallets
+        ^-  (list card)
+        [(btc-node-store-poke /list [%list-wallets ~])]~
+      ::
+          :: %load-wallet
+          :: %lock-unspent
+          :: %remove-pruned-funds
+          :: %rescan-blockchain
+          :: %send-many
+          :: %send-to-address
+          :: %set-hd-seed
+          :: %set-label
+          :: %set-tx-fee
+          :: %sign-message
+          :: %sign-raw-transaction-with-wallet
+          :: %unload-wallet
+          :: %wallet-create-fundedpsbt
+          :: %wallet-lock
+          :: %wallet-passphrase
+          :: %wallet-passphrase-change
+          :: %wallet-process-psbt
+  ==  ==
+::
+++  btc-node-store-poke
+  |=  [pax=path act=btc-node-store-action]
+  ^-  card
+  :*  %pass   pax
+      %agent  [our.bowl %btc-node-store]
+      %poke   [%btc-node-store-action !>(act)]
+  ==
+::
+++  default-wallet
+  =-  .^(@t %gx -)
+  %+  weld  /(scot %p our.bowl)/btc-node-store
+  /(scot %da now.bowl)/default-wallet/noun
```

Comment:
Fun fact, `.^` is n-ary, but boils down to two arguments. It can do list building for you, in that way. Sort of like a `:*`. ie:

```hoon
.^  @t
  %gx
  (scot %p our.bowl)
  %btc-node-store
  (scot %da now.bowl)
  /default-wallet/noun
==
```